frontend/src/components/dashboard/create/email/EmailTemplate.tsx

```diff
@@ -70,13 +72,19 @@ const EmailTemplate = ({
         value={subject}
         onChange={setSubject}
       />
-      <h4>Message</h4>
+      <h4>{protect ? 'Message A' : 'Message'}</h4>
       <p>
         To personalise your message, include keywords that are surrounded by
         double curly braces. The keywords in your message template should match
         the headers in your recipients CSV file.
         <br />
-        <b>Note:</b> Recipient is a required column in the CSV file.
+        <b>Note:</b> Recipient is a required column in the CSV file.{' '}
+        {protect && (
+          <>
+            Include <b>{'{{ protectedlink }}'}</b> to insert the unique
```

Comment:
wording has to imply that protectedlink is a compulsory field. handleSaveTemplate should check for this as well..(along with server-side checks)